src/translators/tests/test_insert_batches.py

```diff
@@ -64,7 +64,7 @@ def generate_insert_payload(self) -> [dict]:
 # then split into 6 batches.
 
 
-class TestDriver:
+class DriverTest:
```

Comment:
hahahaha, yep, it definitely looks like it, since it doesn't follow the XXXTest pattern that tools expect---I'm still waiting for that tool that'll be able to actually understand natural language like a human can. anyhoo, glad to know nothing's broken, phew :-)

Reply:
I believe, within a (pytest-based) test suite, all symbols prefixed with `test_` or `Test` are used as test cases. So, it is perfectly valid use a different name here for a utility class which is not a test case.

Reply:
sure, let's not fight the tool :-)